app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -1,7 +1,7 @@
 .ModuleSearch {
   position: absolute;
   left: 32px;
-  top: 10%;
+  top: calc(50% - 352px + 64px + 80px); /* center - my height + top header + bottom header */
```

Comment:
>  my height 

Will this work when modulesearch has resizable height?

Reply:
Nope. I couldn't find a way to do centering in a nice way. `react-draggable` will overwrite any css transforms.

To make custom heights work we could calculate the position in react code by using window dimensions.

Reply:
TODO